pkg/k8sutils/job.go

```diff
@@ -329,6 +329,10 @@ func (k8s *K8sImpl) CreateK8sJob(
 									Name:  "GIT_COMMIT_ID",
 									Value: jobDetails.GitCommitID,
 								},
+								{
+									Name:  "CONFIGURATION_SERVICE",
```

Comment:
I think that variable is unused in the init container and if not it should probably be renamed to `RESOURCE_SERVICE`

Reply:
Has already been removed ([see](https://github.com/keptn-contrib/job-executor-service/pull/351#discussion_r958417502))